concepts/sbtc/operations/deposit.md

```diff
@@ -1,42 +1,25 @@
 # sBTC Deposit Operation
 
-The deposit operation allows users to convert their BTC to sBTC. This process involves moving BTC from the Bitcoin blockchain to the Stacks blockchain as sBTC.
+The deposit operation enables users to convert their BTC to sBTC, effectively moving value from the Bitcoin blockchain to the Stacks blockchain. This process can be completed within a single Bitcoin block, streamlining the user experience.
```

Comment:
Is it really moving value though? or just adding value? 
i think there needs to be some qualifiers about "a single bitcoin block" as well - single block confirmations are inherently dangerous, and bitcoin can and does have forks (meaning a single block confirmation may be reverted).

Reply:
@AshtonStephens where are we with single Bitcoin block confirmations in regards to potential forks? I remember briefly having this conversation with you but can't remember why we said single block confirmations for deposits.